src/schemas/v2/combined.json

```diff
@@ -19633,41 +19977,67 @@
                 }
               }
             },
-            "centralHeating": {
+            "heating": {
+              "ntsRef": "B3.4",
               "baspiRef": "A7.4",
               "ta6Ref": "12.3",
               "rdsRef": "Buildings|BuildingSections/isUnheated=no|yes|true|false",
               "piqRef": "A7.3",
-              "title": "Central Heating",
+              "title": "Heating",
               "type": "object",
-              "baspiRequired": ["hasCentralHeating"],
+              "baspiRequired": ["heatingType"],
+              "ta6Required": ["heatingType"],
+              "ntsRequired": ["heatingType", "otherHeatingFeatures"],
               "properties": {
-                "hasCentralHeating": {
+                "heatingType": {
+                  "ntsRef": "B3.4.1",
                   "baspiRef": "A7.4.0",
                   "ta6Ref": "12.3.1",
                   "piqRef": "A7.3.1",
-                  "title": "Is there full/partial central heating in your property?",
+                  "title": "What type of heating system does the property have?",
                   "type": "string",
-                  "enum": ["Yes", "No"]
+                  "enum": ["Central heating", "Communal heating system", "None"]
+                },
+                "otherHeatingFeatures": {
+                  "ntsRef": "B3.4.2",
+                  "title": "What other heating features does the property have?",
+                  "type": "array",
+                  "items": {
+                    "type": "string",
+                    "enum": [
+                      "Air Conditioning",
+                      "Double Glazing",
+                      "Eco-Friendly",
+                      "Night Storage",
+                      "Solar Water",
+                      "Undefloor Heating",
+                      "Wood Burner",
+                      "Open Fire",
+                      "Ground Source Heat Pump",
+                      "Air Source Heat Pump",
+                      "Solar Thermal"
+                    ]
+                  }
```

Comment:
What are your thoughts on making this a string for free text with a description rather than an array? My thinking is that the answer to this is not exhaustive. I'm not precious, though this is a strong list.